Tests/xcprojTests/PBXRezBuildPhaseSpec.swift

```diff
@@ -22,7 +22,7 @@ final class PBXRezBuildPhaseSpec: XCTestCase {
     }
 
     func test_equals_returnsTheCorrectValue() {
-        let another = PBXResourcesBuildPhase(files: ["123"],
+        let another = PBXRezBuildPhase(files: ["123"],
```

Comment:
This is an example of something that was incorrectly passing tests

Reply:
:/ we need Sourcery